packages/mineral-ui-tokens/tokens/aliases.json

```diff
@@ -3,10 +3,11 @@
     "./palette/aliases.json"
   ],
   "aliases": {
-    "fontSize_base":  "1rem",
-    "fontSize_mouse": "0.6875rem",
-    "fontSize_prose": "1rem",
-    "fontSize_ui":    "0.875rem",
+    "fontSize_base":      "1rem",
+    "fontSize_mouse":     "0.6875rem",
+    "fontSize_prose":     "1rem",
+    "fontSize_ui":        "0.875rem",
+    "fontSize_custom":    "0.75rem",
```

Comment:
Where is this actually used?  Let's have a more descriptive name than "custom" based on where it's used.